### PR TITLE
repository cleanup

### PR DESCRIPTION
**Pull request by birnbaum**
Includes:
- Cleaning up CI workflows and updating badge in readme
- Deleting unused files
- Some minor cleanup in other files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
 
 # PCAP files from tests
 *.pcap
 
 # Simulation log files
-/simulation-logs
-/examples/simulation-logs
-/docker/sawtooth-pbft/keys
+simulation-logs
+examples/simulation-logs
```
Comment (beilharz) [inline .gitignore:26]: diese Zeile wird so jedenfalls auch schon durch die vorherige unnötig gemacht...

File: .gitignore [hunk 2]
```diff
@@ -1,29 +1,31 @@
 var/*
-.idea/*
+.idea
+.vscode
+
 
 # NS-3 Directory
-/ns3
+ns3
```
Comment (beilharz) [inline .gitignore:7]: ich würde nicht alles, was irgendwo ns3 heißt ignoren

oder hab ich hier ein falsches Verständnis von der Bedeutung der / am Anfang?

**Review by beilharz — COMMENTED** [review]
warum wolltest du den pull_request-opened workflow löschen?

**Comment by birnbaum** [comment]
> This looks great. Two questions/comments:
> 
> * Why did you delete the pull_request-opened workflow? Might not be necessary, but its nice if the link is there, no?

you are right, I'll re-add the pull_request-opened workflow for now! However, after checking the pipelines I'm convinced that we should move all docs to readthedocs: there is so much code related only to building and pushing docs and everything is pretty unstable (yesterday one of my pipeline runs failed bc someone was building something on a different branch at the same time).

> * I think we shouldn't remove the leading slashes everywhere in the .gitignore file – sometimes it makes sense to only ignore in the root-dir.

I can re-add the `/ns3` one but the rest seems pretty fair to exclude, don't you think? E.g. `.virtualenv` or `docs/build` is really common stuff to exclude, also from all subdirectories

**Review by beilharz — APPROVED** [review]

**Pull request merged**